src/wallet/spend.cpp

```diff
@@ -153,6 +153,31 @@ BResult<CoinsResult> FetchSelectedInputs(const CWallet& wallet, const CCoinContr
     return result;
 }
 
+BResult<CoinsFund> FetchCoins(const CWallet& wallet, const CCoinControl& coin_control,
```

Comment:
In 84b4ffe85ba0267c49080e9a096e45d6af63e720 "wallet: remove fetch pre-selected-inputs responsibility from SelectCoins"

I don't think it's necessary to have another function here to wrap these two calls. They could just be in `CreateTransactionInternal`

Reply:
True. It's not used anywhere else for now.
Have removed the function and the `CoinsFunds` struct as well.